### ISSUE TITLE
visible attribute has no effect on gradio.Group()

### ISSUE THREAD

**Issue by shahbuland** [gradio-app/gradio#2246]
### Describe the bug

- Setting visible = false in gr.Group (i.e. with gr.Group(visible = False)) does nothing
- Subbing in gr.Column instead of gr.Group works as expected with regards to visibility
- gr.update(visible = False) also has no effect on gr.Group

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

```python
import gradio as gr

with gr.Blocks() as demo:
    with gr.Group(visible = True):
        gr.Textbox("A")
    with gr.Group(visible = False):
        gr.Textbox("B")
```

### Screenshot

[Running the demo provided in reproduction code](https://cdn.discordapp.com/attachments/738882678711123980/1019013634636124210/unknown.png)

### Logs

```shell
No errors so no logs
```

### System Info

```shell
Browser = Chrome
gradio version = 3.3
```

### Severity

serious, but I can work around it

**Comment by aliabid94** [comment]
will take a look!

**Comment by lcrh** [comment]
The buggy behavior has reappeared as of '3.34.0'

![image](https://github.com/gradio-app/gradio/assets/14020730/5bb8e6b0-30a4-4451-9d7c-6322dbddb022)

**Comment by taesiri** [comment]
Any update on this?

**Comment by taesiri** [comment]
We have a fix for this here #4567
